ra/1.1/resource-agent-api.md

```diff
@@ -344,148 +410,264 @@ Currently, the following additional environment variables are defined:
+<https://github.com/ClusterLabs/OCF-spec/blob/master/ra/1.1/ra-api.rng>.
 
-### Semantics
+### Example
 
-An example of a valid meta data output is provided in
-`ra-metadata-example.xml`.
+An example of a valid meta-data output is provided at
+<https://github.com/ClusterLabs/OCF-spec/blob/master/ra/1.1/ra-metadata-example.xml>.
 
-## To-do list
+### Semantics
 
-- Move the terminology definitions out into a separate document
-  common to all OCF work.
-- An interface where the RA asynchronously informs the RM of
-  failures is planned but not defined yet. 
+Certain meta-data XML elements warrant further explanation:
+
+- `resource-agent`: The optional `version` attribute should describe the
+  version of the agent itself.
+
+- `version`: This is the version of the OCF RA standard with which the RA
+  claims compatibility.
+
+- `longdesc`, `shortdesc`, and `desc`: These elements, wherever they occur in
+  meta-data, are natural-language descriptions of what is specified by their
+  parent element, intended as hints to tools for display to users. These
+  elements must contain a `lang` attribute whose value is a standard language
+  identifier ("BCP 47" <https://www.rfc-editor.org/rfc/bcp/bcp47.txt>).
+  Multiple elements with different values for `lang` may be specified, to
+  provide translations. These elements may contain any XML, but it is strongly
+  recommended to limit the content to a text string.
+
+- `parameter`:
+    - `unique-group` attribute: This is a hint to RMs and other tools that the
+      combination of all parameters with the same value should be unique to the
+      resource type. That is, no two instances of the same resource type should
+      have the same combination of these parameters. If not specified, multiple
+      instances of the same resource type may have the same value of this
+      parameter.
+
+      Note: a boolean `unique` attribute was formerly used for a similar
+      purpose, but was commonly misused, and is now deprecated. An RA may
+      provide it for backward compatibility, but because it is only a hint in
+      any case, RMs and tools are free to ignore it.
+
+    - `required` attribute: This is a hint to RMs and other tools that every
+      resource instance of this resource type must specify a value for this
+      parameter.
+
+    - `reloadable` attribute: If set to 1, this is a hint to indicate that a
+      change in this attribute does not necessitate a full stop and start to
+      take effect, but can take effect via a `reload-params` action.
+
+    - `deprecated` child element: When present, this element is a hint to RMs
+      and other tools that the parameter is supported for backward
+      compatibility only.
+      - `replaced-with` child element: This must contain a `name` attribute
+        with the name of another parameter that should be used instead of the
+        deprecated parameter. Multiple such elements maybe specified.
```

Comment:
s/maybe/may be/

Reply:
Updated "corrections" commit with this fix (and carried it over to 1.1 in updated "bring in proposed changes" commit)